bindings/r/R/update.R

```diff
@@ -0,0 +1,30 @@
+# Assumes that you have forked tree-sitter-r in its entirety, but your
+# current working directory is `tree-sitter-r/bindings/r/`, as set by the
+# `.Rprofile` in the root of the directory.
+update <- function() {
```

Comment:
Perhaps instead this could be done in the package `configure` script? Something like `--with-treesitter-grammar=/path/to/grammar`, with some support for automatically inferring the appropriate location? Or it could be downloaded on-the-fly using some appropriate git commit?

Reply:
(if you like the idea but hate writing `configure` scripts, you might like https://github.com/kevinushey/configure)

Reply:
It's quite hard to get this right for all of:
- Sending a release to CRAN
- Local development with `load_all()`
- CI
- Installing from github

So for now I'm going to continue manually updating it, but now `load_all()` is at least smart enough to automatically update when things look out of sync, and I think that will be pretty helpful